source/developers/component_loading.markdown

```diff
@@ -13,6 +13,9 @@ A component will be loaded on start if a section (ie. `light:`) for it exists in
 
  * `<config directory>/custom_components/<component name>`
  * `homeassistant/components/<component name>` (built-in components)
+    
+    If Home Assistant is installed in an **venv**:
+    * `homeassistant/lib/python3.5/site-packages/homeassistant/components/<component name>`
```

Comment:
That path is mentioned already on line 15. Also, the path for a `venv` will most likely be completely different than the one you provided..

Reply:
I installed Home Assistant according to the introductions, so the full path is: `/srv/homeassistant/lib/python3.5/site-packages/homeassistant/components/<component name>` why do you think the path will be completely different?

Maybe on the component loading page should be an notice that the homeassistant base folder is inside the `venv`? Or is that too obvious?

Reply:
Because the [`venv` guide](https://home-assistant.io/docs/installation/virtualenv/) doesn't suggest a location. The Raspbian guide uses `/srv/homeassistant`. I use `/opt/homeassistant` and `~/home/repos/home-assistant` that one because I don't have to deal with permissions. 

> Or is that too obvious?

It seems not. Otherwise you wouldn't have to suggest an adjustment. Let's add a note that about a possible location.